tests/integration/config/plugins/integration-tests/data/tests/tests_common.txt

```diff
@@ -539,6 +539,120 @@ test "Load First Savegame"
 			key l
 
 
+test "Create New Pilot"
+	status partial
+	description "Helper function to create a new pilot."
+	sequence
+		input
+			key l
+		# new pilot
+		input
+			key n
+		# start
+		input
+			key s
+		# first name: a
+		input
+			key a
+		input
+			key "return"
+		# last name: b
+		input
+			key b
+		input
+			key "return"
+		# skip text
+		input
+			key "return"
+		# go to the main landing panel
+		input
+			key l
+
+
+test "Buy First Ship"
+	status partial
+	description "Helper function to buy the first ship of the shipyard, starting from the planet panel."
+	sequence
+		input
+			key s
+		# buy first ship
+		input
+			key "down"
+		input
+			key b
+		# confirm
+		input
+			key "return"
+		# back to landing
+		input
+			key l
+
+
+test "Wait 55 steps"
+	status partial
+	description "Wait 55 steps for something that requires some time, like launching or loading the game."
+	sequence
+		apply
+			"test: wait 55" = 55
+		label wait_load
+		apply
+			"test: wait 55" -= 1
+		input
+		branch wait_load
+			"test: wait 55" > 0
+
+
+test "Wait 10 steps"
+	status partial
+	description "Wait 10 steps to give time for the game to get the newest conditions. 5 steps are just enough but may as well be sure."
+	sequence
+		apply
+			"test: wait 10" = 10
+		label wait
+		apply
+			"test: wait 10" -= 1
+		input
+		branch wait
+			"test: wait 10" > 0
+
+
+test "Start Game"
+	status active
+	description "Test if one can start the game correctly: choose a name and buy a ship."
+	sequence
+		watchdog 1000
+		call "Create New Pilot"
+		# the game needs a few steps to load the new pilot
+		call "Wait 55 steps"
+		call "Buy First Ship"
+		# game needs time to process showing the mission
+		call "Wait 10 steps"
+		# beat it, old timer
+		input
+			key 3
+		input
+			key "return"
+		# to be sure
+		input
+			key l
+		# normally the mission being launched means the conditions are refreshed (or they always are?) and we don't need to launch
+		# but to be sure:
+		call "Depart"
```

Comment:
This should no longer be needed if all of the conditions below are correctly implemented as automatic on-demand conditions.

Reply:
Well thats the thing, I'm pretty sure I tested it and those 5 inputs are needed for some reason

Reply:
Eh you know what lets see
```suggestion
```